README.md

```diff
@@ -29,5 +29,10 @@ Import-Module -Name '.\Carbon.Windows'
 ## Commands
 
 * `Get-CBackConnectionHostName`, for getting the list of currently configured back connection hostnames.
+* `Get-CEventLog`, as a drop in replacement for using `Get-EventLog` in PowerShell 7.
+* `Install-CEventLog`, as an idempotent version of `Get-CEventLog`.

```

Comment:
Rephrase to `Install-CEventLog which creates an event log only if it doesn't already exist.`. Ditto for `Uninstall-CEventLog`.